### DISCUSSION TITLE
Can you configure which device is the console?

### DISCUSSION THREAD

**Discussion by sparkybrit** [wwarthen/RomWBW#295] [Q&A] [answered]
Can you configure which serial device is the console (i.e. the device that is assigned to be unit 0)?

**Comment by feilipu** [comment, marked as answer]
Yes, as done [here](https://github.com/feilipu/ux_module/blob/main/README.md#romwbw) to ensure the ACIA (UX Module emulation) is the console. Specify `BOOTCON` to be the desired console unit.

**Comment by wwarthen** [comment]
Note that the character device unit numbers are assigned dynamically at boot time depending on which hardware is discovered.  So, you may need to update BOOTCON if your hardware configuration changes.
